moneta/moneta/vaextended/bqplot.py

```diff
@@ -390,6 +402,22 @@ def update_zoom_brush_vertical(self, *args):
                     self.zoom_brush.selected = None
                 self.zoom_sel(None, None, y1, y2, smart_zoom=False, padding=False)
 
+    def measure_selected_area(self, *args):
+        with self.output:
+            if not self.ruler.brushing: # Update on mouse up
+                pass
+            if self.ruler.selected is not None:
+                (x1, y1), (x2, y2) = self.ruler.selected
+                if not self.ruler.brushing: # Update on mouse up
+                    self.figure.interaction = self.ruler
+                with self.ruler.hold_trait_notifications(): # Delete selection
+                    self.ruler.selected_x = None
+                    self.ruler.selected_y = None
+                df = self.get_df_selection(x1, x2, y1, y2)
+                uniqueCacheLines = {l >> int(np.log2(self.plot.model.curr_trace.cache_block)) for l in df.unique('Address')}
+                self.plot.model.selected_values = (y2-y1, len(uniqueCacheLines), df.count())
```

Comment:
Why do you need to pass the selected value all the way up to the model? I think it would make sense for this value to be directly passed to measurement.update()

Reply:
I've fixed this. I was initially confused about the structure of these classes and saw we pass trace information in that way.

Reply:
Can you remove self.plot.model.selected_values if you are not using it.

Reply:
Fixed.